cpp/ql/src/semmle/code/cpp/controlflow/internal/CFG.qll

```diff
@@ -983,36 +981,65 @@ private predicate subEdgeIncludingDestructors(Pos p1, Node n1, Node n2, Pos p2)
   // called, connect the "before destructors" node directly to the "after
   // destructors" node. For performance, only do this when the nodes exist.
   exists(Pos afterDtors | afterDtors.isAfterDestructors() | subEdge(afterDtors, n1, _, _)) and
-  not exists(getDestructorCallAfterNode(n1, 0)) and
+  not synthetic_destructor_call(n1, 0, _) and
```

Comment:
Why use the raw db table here when it's wrapped by `getSynthesisedDestructorCallAfterNode`?